io/src/main/scala/sbt/io/PathMapper.scala

```diff
@@ -97,16 +100,16 @@ abstract class Mapper {
    * Selects all descendants of `base` directory and maps them to a path relative to `base`.
    * `base` itself is not included.
    */
-  def allSubpaths(base: File): Traversable[(File, String)] =
+  def allSubpaths(base: File): Vector[(File, String)] =
```

Comment:
Actually, here we may not want a Vector.  "allSubpaths" may want to be a lazy traversable rather than something which keeps all the file objects in memory.

remember the bugs about "running sbt in `/` cause out of memory exception"?   This is an instance where you don't want vector.